main.tf

```diff
@@ -124,7 +124,7 @@ resource "azurerm_kubernetes_cluster" "main" {
 
 resource "azurerm_log_analytics_workspace" "main" {
   count               = var.enable_log_analytics_workspace ? 1 : 0
-  name                = "${var.prefix}-workspace"
+  name                = var.cluster_name == "" ? "${var.prefix}-workspace" : var.cluster_name
```

Comment:
Could we add a variable as `var.cluster_workspace_name` ?

Reply:
I am adding the 'cluster_analytics_workspace' variable. Do you agree with this name?